lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -1223,13 +1223,12 @@ fini(void)
 static void *
 mrs_malloc(size_t size)
 {
-#ifdef JUST_INTERPOSE
```

Comment:
I do wonder if we even still need `JUST_INTERPOSE`?  Presumably it was useful as a benchmark baseline, but probably not useful going forward?

Reply:
I've removed it and actually realized we should remove all the `JUST_` ifdefs so I'll do that shorty.